images/vault/vault.tf

```diff
@@ -0,0 +1,9 @@
+module "vault-config" { source = "./configs/vault" }
+
+module "vault" {
+  source            = "../../tflib/publisher"
+  name              = basename(path.module)
+  target_repository = var.target_repository
```

Comment:
is it correct? or should we have `-vault` suffix here?

Reply:
This is perfect otherwise it is `vault-vault`